recipes/twitch-native-ipc/all/patches/v3.1.1.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 491ae46..c0a4f3d 100644
+--- a/CMakeLists.txt
```

Comment:
I think you should also patch the install command for Windows:
see https://github.com/twitchtv/twitch-native-ipc/blob/81154497fcf55288c51cb5c93e95014ec4f0644d/libnativeipc/CMakeLists.txt#L84

The ARCHIVE and RUNTIME component should be installed to lib and bin respectively.